app/main.py

```
@@ -7,16 +8,48 @@
+from slowapi.errors import RateLimitExceeded
+from slowapi.middleware import SlowAPIMiddleware
+
+# Custom rate limiting exceptions
+from app.core.rate_limiting.exceptions import (
+    RateLimitExceededException,
+    CircuitBreakerOpenException,
+    GeminiServiceUnavailableException
+)
+
 from app.agents_v2.orchestration.graph import app as agent_graph
 from app.agents_v2.orchestration.state import GraphState # Corrected import location
```

Ruff (0.12.2)

`app.agents_v2.orchestration.state.GraphState` imported but unused

Remove unused import: `app.agents_v2.orchestration.state.GraphState`

(F401)